spec/06-expressions.md

```diff
@@ -329,8 +329,8 @@ arguments, the following conditions must hold.
 - The names $x_i$ of all named arguments are pairwise distinct and no named
   argument defines a parameter which is already specified by a
   positional argument.
-- Every formal parameter $p_j:T_j$ which is not specified by either a positional
```

Comment:
Yes, there is no notion of implicit arg here. For another ticket, I was just rereading 6.6 because I'd forgotten what a "function application" is. :1st_place_medal: It is so many things.

Maybe the note should go in 4.6.1 after all; I only just noticed that there is another clarification there about scoping (which bears on another ticket).

Basically, that can be the "what happens if I leave out this arg?" FAQ.

For some reason, during an insomniac night, I thought over here in func apps was the natural place to describe semantics of `f()`.